src/main/scala/com/cloudera/sparkts/TimeSeries.scala

```diff
@@ -169,6 +169,55 @@ class TimeSeries[K](val index: DateTimeIndex, val data: DenseMatrix,
   }
 
   /**
+    * Returns a TimeSeries where each individual time series is differenced with the given time
+    * frequency.
```

Comment:
Also, note that I just tested this example and there is a bug: the function currently returns:

t ------ d(value)
10pm 1.1
11pm 2.4

The value at 10pm shouldn't be there, since between 1pm and 10pm there hasn't been 10 minutes elapsed... Will look into this.